chromium_src/chrome/grit/theme_resources.h

```diff
@@ -0,0 +1,2 @@
+#include "brave/grit/brave_theme_resources.h"
+#include "gen//chrome/grit/theme_resources.h"
```

Comment:
I think this should be one slash. no big deal just want to keep it clean.

Reply:
Done.